src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -441,12 +488,13 @@ pal::string_t fx_muxer_t::resolve_fx_dir(host_mode_t mode, const pal::string_t&
 
         if (!do_roll_forward)
         {
-            trace::verbose(_X("Did not roll forward because specified version='%s', patch_roll_fwd=%d, chose [%s]"), specified_fx_version.c_str(), config.get_patch_roll_fwd(), fx_ver.c_str());
+            trace::verbose(_X("Did not roll forward because specified version='%s', patch_roll_fwd=%d, roll_fwd_on_no_candidate_fx=%d, chose [%s]"),
+                specified_fx_version.c_str(), config.get_patch_roll_fwd(), roll_fwd_on_no_candidate_fx_enabled, fx_ver.c_str());
             append_path(&fx_dir, fx_ver.c_str());
         }
         else
         {
-            resolve_roll_forward(fx_dir, fx_ver, specified);
+            resolve_roll_forward(fx_dir, fx_ver, specified, config.get_patch_roll_fwd(), roll_fwd_on_no_candidate_fx_enabled);
```

Comment:
You can factor out config.get_patch_roll_fwd and use its value instead of invoking twice (here and above).